### ISSUE TITLE
Export Invalid JSON

### ISSUE THREAD

**Issue by garywoodfine** [DataTurks/DataTurks#2]
When downloading a completed Data Annotation Task, with mutliple files included, the JSON file is not valid file because it is not defined as an array and is missing comma's between array entities.
 
i.e
` { "content" : "blah ..." .... }  {"content" : "blah ..." ...}`

should be

`[ { "content" : "blah ..." .... } ,  {"content" : "blah ..." ...} ] `

It makes it very difficult to ingest the file otherwise because it breaks Valid Json Formats and when working on large Annotation projects the files are large and very difficult to patch.

**Comment by gajju3588** [comment]
As input file is generally new line seperated, output is also new line seperated. 

It was very hard to come to a common ground regarding this, but i guess you can add one more output format which will have a json array.

**Comment by garywoodfine** [comment]
In my opinion exporting a valid JSON array format would be more compliant as most developers will be importing the output of this phase using JSON Parsers  and modeling the classes on the JSON.  At least it is the process I used.

Exporting the from the web app, and importing the contents to another database in order to build the model.  In my case, we needed to transform the output to an input that is accepted by AWS Comprehend.
